api/v1beta1/foundationdbcluster_types_test.go

```diff
@@ -2329,15 +2329,28 @@ var _ = Describe("[api] FoundationDBCluster", func() {
 					input: "127.0.0.1:bad",
 					err:   fmt.Errorf("strconv.Atoi: parsing \"bad\": invalid syntax"),
 				}),
-			Entry("IPv6 with invalid address",
+			Entry("host name",
 				testCase{
-					input: "[::1:]:4500",
```

Comment:
Can we leave those tests in place? Those are still invalid IP addresses and invalid hostnames.

Reply:
I think there were some issues with trying to validate the IP addresses when it needs to be able to accept host names as well, but I'll take another look at it.

Reply:
Because the process address can contain a DNS name in some contexts, we can't apply the same validations. We can introduce new validations to ensure it's a valid DNS name, but I don't know if that's worth the complexity.

Reply:
I think we don't want to have that complexity in our code we might want to leverage some validation features from the kube api-machinery for that.